www/lobby/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Router, Request, Response } from 'express';
+import { Lobby } from '../../lib/lobby';
+import { User } from '../../lib/user';
+import { validateJwt } from '../../utils/jwt';
+import { generateLobbyId } from '../../utils/lobby';
+import { lobby_id_router } from './:id';
+
+export const lobby_router = Router();
+
+lobby_router.use('/midtest', validateJwt);
```

Comment:
? whats this for?

Reply:
Oh its supposed to be .use('/', validateJwt) to validate all the routes but it makes testing more complicated so i turned  it off

Reply:
ah i see